lib/iris/util.py

```diff
@@ -1211,6 +1211,43 @@ def add_coord(coord):
     return new_cube
 
 
+def cube_squeeze(cube):
```

Comment:
Could just call this `squeeze`, its presence in iris indicates it operates on cubes (as does the docstring).